charts/backstage/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.7.0
+version: 0.7.1
```

Comment:
Can you please change this to `0.8.0` instead? This PR implements a new feature, not a fix.

Reply:
done